app/cdap/components/RecipeSteps/IconStore/InfoGraphicData.tsx

```diff
@@ -0,0 +1,208 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+import React from 'react';
+
+export const InfoGraphicData = (
```

Comment:
We should review the usage of this background graphic.

Reply:
as per discussion with design team the updated icon we will get on  29th November  once we get that we will update it accordingly

Reply:
updated icon already given for review to the google team, once we get feedback will update it accordingly